src/common.speech/ServiceRecognizerBase.ts

```diff
@@ -278,6 +279,9 @@ export abstract class ServiceRecognizerBase implements IDisposable {
 
     public sendMessage(message: string): void { }
 
+    public set activityTemplate(messagePayload: string) { this.privActivityTemplate = messagePayload; }
+    public get activityTemplate(): string { return this.privActivityTemplate; }
```

Comment:
Is this a base class thing?

Reply:
I set as such since we want access from the api surface as well as the recognizer, but as I "say" that out loud perhaps this is better served via the property collection?